futurex_openedx_extensions/dashboard/details/learners.py

```diff
@@ -154,18 +158,35 @@ def get_learners_queryset(
     :return: QuerySet of learners
     :rtype: QuerySet
     """
-    tenant_sites = get_tenants_sites(fx_permission_info['permitted_tenant_ids'])
+    full_access_tenant_ids = set()
+    for org in fx_permission_info['view_allowed_full_access_orgs']:
+        full_access_tenant_ids.update(get_org_to_tenant_map()[org])
+    full_access_tenant_ids = full_access_tenant_ids.intersection(fx_permission_info['permitted_tenant_ids'])
```

Comment:
`&` is shorthand for intersection:

```suggestion
    full_access_tenant_ids = full_access_tenant_ids & fx_permission_info['permitted_tenant_ids']
```